sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -73,6 +73,8 @@ class AioHttpTransport(AsyncHttpTransport):
             :caption: Asynchronous transport with aiohttp.
     """
     def __init__(self, *, session: Optional[aiohttp.ClientSession] = None, loop=None, session_owner=True, **kwargs):
+        if loop and sys.version_info >= (3, 10):
+            raise ValueError("Starting Python 3.10, asyncio doesn’t support loop as a parameter anymore")
```

Comment:
Do we want to add a deprecation warning `if loop and sys.version_info < (3, 10)`?

Reply:
Grammar nitpick: Starting **with** Python 3.10, asyncio no longer supports the loop parameter.

Reply:
No. If user does not use Python >= 3.10, we want to behave exactly same. There should be no impact at all. (new warnings are still side effects.)